MachOKit/Segments/MKSegment.m

```diff
@@ -198,6 +198,7 @@ - (instancetype)initWithLoadCommand:(id<MKLCSegment>)segmentLoadCommand error:(N
             }
         }
         
+        [MKBackedNode sortOptionalNodeArray:(NSMutableArray *)segmentSections];
```

Comment:
Like the segments array, I prefer to keep the sections array matching the order of the load command.

Reply:
Handled in bcac3dc, similarly to the change in `MKMachO+Segments.m`